packages/node/src/build.ts

```diff
@@ -78,14 +80,28 @@ async function downloadInstallAndBundle({
     meta
   );
   const spawnOpts = getSpawnOptions(meta, nodeVersion);
-  await runNpmInstall(
-    entrypointFsDirname,
-    [],
-    spawnOpts,
-    meta,
-    nodeVersion,
-    config.projectSettings?.createdAt
-  );
+
+  const installCommand = config.projectSettings?.installCommand;
+  if (typeof installCommand === 'string' && considerBuildCommand) {
+    if (installCommand.trim()) {
+      console.log(`Running "install" command: \`${installCommand}\`...`);
+      await execCommand(installCommand, {
+        ...spawnOpts,
+        cwd: entrypointFsDirname,
+      });
+    } else {
+      console.log(`Skipping "install" command...`);
```

Comment:
Are we advertising `"install": ""` as a way to skip install? If so a small explanation of why might be good.

Reply:
I don't know of the use case, just copied from other places. I think the logic here is just to we don't do `exec("")` but logging as a convention for what the others do 🤷